mino/minogrpc/mod_test.go

```diff
@@ -62,7 +62,7 @@ func TestMinogrpc_FailGenerateKey_New(t *testing.T) {
 	require.EqualError(t, err, fake.Err("overlay: cert private key"))
 }
 
-func TestMinogrpc_FailCreateCert_New(t *testing.T) {
+func TestMinogrpc_FailCreateCerTNew(t *testing.T) {
```

Comment:
Why did you put the `T` here capital? Has it something to do with the `T` faulty ones? To me it seems it's part of `certificates`, no?